source/sigv4.c

```diff
@@ -677,8 +738,8 @@ static SigV4Status_t getCredentialScope( SigV4Parameters_t * pSigV4Params,
 
 /*-----------------------------------------------------------*/
 
-    static int cmpKeyValue( const void * pFirstVal,
-                            const void * pSecondVal )
+    static int cmpField( const void * pFirstVal,
```

Comment:
MISRA requires us to always specify the bit-width of integral types like `int`
```suggestion
    static int32_t cmpField( const void * pFirstVal,
```